bundles/framework/admin-users/view/RolesTab.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { TextInput, Button, Confirm, Message } from 'oskari-ui';
+import { PrimaryButton } from 'oskari-ui/components/buttons';
+import { DeleteOutlined } from '@ant-design/icons';
+import styled from 'styled-components';
+
+const Content = styled('div')`
+    display: flex;
+    flex-direction: column;
+`;
+
+const Form = styled('div')`
+    display: flex;
+    flex-direction: row;
+    margin-bottom: 20px;
+    align-self: flex-end;
+    align-items: center;
+`;
+
+const AddButton = styled(PrimaryButton)`
+    width: 50px;
+    margin-left: 15px;
+    align-self: flex-end;
+`;
+
+const DeleteButton = styled(Button)`
```

Comment:
We have a delete button with confirm wrapping in oskari-ui buttons already. We should use that instead since it's nice to have a confirm on delete and it adds a lot of noise that we don't need here if we use the common component.